pkg/resourcemanager/controller/node/reconciler.go

```diff
@@ -158,6 +177,63 @@ func AllNodeCriticalPodsAreReady(log logr.Logger, recorder record.EventRecorder,
 	return true
 }
 
+// GetRequiredDrivers searches through the pods annotations, and returns a set
+// of driver names if it finds annotations with the wait-for-csi-node prefix;
+// otherwise it returns an empty set.
+func GetRequiredDrivers(pods []corev1.Pod) sets.Set[string] {
+	requiredDrivers := sets.Set[string]{}
+	for _, pod := range pods {
+		drivers := []string{}
```

Comment:
```suggestion
		var drivers []string
```

However, why is this needed? Can't you add to `requiredDrivers` directly in L189?